.buildkite/pipeline.yml

```diff
@@ -24,6 +22,7 @@ steps:
       - julia --project=test -e 'using CUDA; CUDA.precompile_runtime()'
       - julia --project=test -e 'using Pkg; Pkg.status()'
     agents:
+      partition: gpu
```

Comment:
I think you can remove this if you want. It should pick up the gpu queue anyway.